### PR TITLE
refactor: JUnit Jupiter best practices

### PR DESCRIPTION
**Pull request by yeikel**
A bit of [dogfooding ](https://en.wikipedia.org/wiki/Eating_your_own_dog_food ) shouldn't hurt :)

### CODE REVIEW

**Comment by yeikel** [comment]
The failure seems unrelated to this change : 


> Could not store entry d7693ea75e7ce7ed2f68b9a26b7de44b in remote build cache
> org.gradle.api.UncheckedIOException: Storing entry at 'https://ge.openrewrite.org/cache/d7693ea75e7ce7ed2f68b9a26b7de44b' response status 403: Forbidden
> 	at org.gradle.caching.http.internal.HttpBuildCacheService.throwHttpStatusCodeException(HttpBuildCacheService.java:174)
> 	at org.gradle.caching.http.internal.HttpBuildCacheService.store(HttpBuildCacheService.java:151)

**Review by timtebeek — APPROVED** [review]

**Comment by timtebeek** [comment]
This is in the logs a bit further up, but I don't see the equivalent code in main:
```
/home/runner/work/rewrite/rewrite/rewrite-gradle/src/test/java/org/openrewrite/gradle/AddGradleWrapperTest.java:39: error: GradleWrapperTest is not public in org.openrewrite.gradle.util; cannot be accessed from outside package
import static org.openrewrite.gradle.util.GradleWrapperTest.mockGradleServices;
                                         ^
/home/runner/work/rewrite/rewrite/rewrite-gradle/src/test/java/org/openrewrite/gradle/AddGradleWrapperTest.java:39: error: static import only from classes and interfaces
import static org.openrewrite.gradle.util.GradleWrapperTest.mockGradleServices;
^
```
Did notice your branch is 166 commits behind main; that's odd when you made this through the platform.. 🤔

**Comment by yeikel** [comment]
> This is in the logs a bit further up, but I don't see the equivalent code in main:
> ```
> /home/runner/work/rewrite/rewrite/rewrite-gradle/src/test/java/org/openrewrite/gradle/AddGradleWrapperTest.java:39: error: GradleWrapperTest is not public in org.openrewrite.gradle.util; cannot be accessed from outside package
> import static org.openrewrite.gradle.util.GradleWrapperTest.mockGradleServices;
>                                          ^
> /home/runner/work/rewrite/rewrite/rewrite-gradle/src/test/java/org/openrewrite/gradle/AddGradleWrapperTest.java:39: error: static import only from classes and interfaces
> import static org.openrewrite.gradle.util.GradleWrapperTest.mockGradleServices;
> ^
> ```
> Did notice your branch is 166 commits behind main; that's odd when you made this through the platform.. 🤔 

Could this be because I already had a fork, and the platform used my main branch without updating it first?

**Comment by timtebeek** [comment]
Turns out the ingestion of this project hasn't been running smoothly lately, leading to outdated AST files, and hence outdated patches produced. 🤔

File: rewrite-gradle/src/test/java/org/openrewrite/gradle/util/GradleWrapperTest.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class GradleWrapperTest {
+class GradleWrapperTest {
```
Comment (timtebeek) [inline rewrite-gradle/src/test/java/org/openrewrite/gradle/util/GradleWrapperTest.java:38]:
```suggestion
public class GradleWrapperTest {
```

Comment (timtebeek) [reply rewrite-gradle/src/test/java/org/openrewrite/gradle/util/GradleWrapperTest.java:38]: Let's see if this is still needed after resolving the conflicts; it it is we can apply this and still see the PR through.

**Pull request merged**